packages/slice-machine/src/features/onboarding/SliceMachineOnboardingGuide/OnboardingProvider.tsx

```diff
@@ -60,14 +60,18 @@ export const OnboardingProvider = ({
 
     if (newCompleteState) {
       void telemetry.track({
-        event: "onboarding:step-completed",
+        event: "shared-onboarding-guide:step-completed",
```

Comment:
❓That's the old guide, no? Should take the old guide tracking key, no?

Reply:
Yeah, you're correct, fixed.